### PR TITLE
[BPK-2731]: Added docs for new nudger component

### PR DESCRIPTION
**Pull request by ojcurtis**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/src/pages/WebNudgersPage/NudgersPage.js [hunk 1]
```diff
@@ -53,6 +59,65 @@ class NudgerContainer extends Component<{ buttonType: ?string }, {}> {
   }
 }
 
+const options = ['economy', 'premium', 'business', 'first'];
```
Comment (shaundon) [inline docs/src/pages/WebNudgersPage/NudgersPage.js]: Could we write these in title case? I think it'd make for a nicer-looking example (you'd need to change lines, 93, 105 and 106 too).

```suggestion
const options = ['Economy', 'Premium', 'Business', 'First'];
```

File: docs/src/pages/WebNudgersPage/NudgersPage.js [hunk 2]
```diff
@@ -76,6 +141,17 @@ const components = [
     dark: true,
     examples: [<NudgerContainer buttonType="outline" />],
   },
+  {
+    id: 'configurable',
+    title: 'Configurable Nudger',
+    blurb: [
+      <Paragraph>
+        A configurable nudger to support non numeric variants. E.g. Dates, text,
```
Comment (shaundon) [inline docs/src/pages/WebNudgersPage/NudgersPage.js]:
```suggestion
        A configurable nudger to support non-numeric variants, for example dates and strings.
```

File: docs/src/pages/WebNudgersPage/NudgersPage.js (hunk 1 above)
Comment (ojcurtis) [reply docs/src/pages/WebNudgersPage/NudgersPage.js]: Done

File: docs/src/pages/WebNudgersPage/NudgersPage.js (hunk 2 above)
Comment (ojcurtis) [reply docs/src/pages/WebNudgersPage/NudgersPage.js]: Done

**Pull request merged**